### PR TITLE
adjustment query select data per week

### PR DESCRIPTION
**Pull request by rachadiannovansyah**
Overview:
- Add condition for sorting
- Adjustment some query by case

cc: @jabardigitalservice/jds-backend

### CODE REVIEW

**Review by khihadysucahyo — APPROVED** [review]

**Review by firmanJS — APPROVED** [review]

File: core-service/src/database/migrations/20210830234412_create_events_table.up.sql [hunk 1]
```diff
@@ -6,8 +6,8 @@ CREATE TABLE `events` (
     `title` varchar(80) NOT NULL,
     `description` varchar(255),
     `date` datetime NOT NULL,
-    `start_hour` int(12) NOT NULL,
-    `end_hour` int(12) NOT NULL,
+    `start_hour` varchar(12) NOT NULL,
+    `end_hour` varchar(12) NOT NULL,
```
Comment (khihadysucahyo) [inline core-service/src/database/migrations/20210830234412_create_events_table.up.sql:10]: dude, if u were edit this migration file, you need to rollback to the previous migration version

**Comment by rachadiannovansyah** [comment]
> I am just wondering, why you used `varchar` and not date time?

after some consideration is a good idea to implement that struct, thanks!

File: core-service/src/database/migrations/20210830234412_create_events_table.up.sql (hunk 1 above)
Comment (khihadysucahyo) [reply core-service/src/database/migrations/20210830234412_create_events_table.up.sql:10]: i mean, you need to migrate down then up and you must considering the version.
do you think by making the alter migration file much better?

Comment (rachadiannovansyah) [reply core-service/src/database/migrations/20210830234412_create_events_table.up.sql]: i also think the same, but I think we're still pre-dev on staging..
or need to create new migration bout these changes?

Comment (khihadysucahyo) [reply core-service/src/database/migrations/20210830234412_create_events_table.up.sql]: That's okay dude, just reminding you to rollback this previous version, before migrating to staging
**prevent to migrate down all

**Pull request merged**